wire-ios-system/Source/ZMSLog.swift

```diff
@@ -245,30 +258,28 @@ extension ZMSLog {
 
 // MARK: - Save on disk & file management
 
-extension ZMSLog {
+public extension ZMSLog {
     private enum Constant {
         static let maxNumberOfLogFiles = 5
     }
 
-    static var cachesDirectory: URL? {
+    internal static var cachesDirectory: URL? {
```

Comment:
How come the need for an _explicit_ `internal` here and in some other places?

Reply:
There is this SwiftFormat rule [extensionAccessControl](https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#extensionAccessControl) and we agreed on having an access modifier next to `extension`. 
In this case the extension is `public`, so any declaration that shouldn't be public needs `internal`.
The formatter did this automatically.

Reply:
Oh yeah, I see! Thank you!